### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by katya-sn**
- [DEMO LINK](https://katya-sn.github.io/layout_stars/)
- [TEST REPORT LINK](https://katya-sn.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,5 +9,56 @@
...
+        <div class="stars star--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="stars star--1">
+          <div class="stars__star stars__star--active"></div>
```
Comment (mykhalenych) [inline src/index.html:23]: Screenshot 2023-08-22 at 07 10 36

missed requirements

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your links aren't working

File: src/index.html (hunk 1 above)
Comment (katya-sn) [reply src/index.html:23]: я неправильно розумію вимоги -  я додала в кожен з 5ти блоків клас stars--N ?

Comment (katya-sn) [reply src/index.html:23]: oops, stars (not star)

File: src/index.html [hunk 2]
```diff
@@ -9,5 +9,56 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="stars stars--2">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:32]: Need to fix it everywhere
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Demo link, you need to make deploy
image

**Comment by katya-sn** [comment]
https://katya-sn.github.io/layout_stars/

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your test link isn't working

**Pull request closed without merging**